crypto/ring-signature/Cargo.toml

```
@@ -6,32 +6,39 @@ edition = "2021"
 readme = "README.md"
 
 [features]
-alloc = ["serde/alloc", "mc-crypto-digestible/alloc"]
-default = ["alloc"]
+alloc = ["serde/alloc", "mc-crypto-digestible/alloc", "mc-util-repr-bytes/alloc", "mc-transaction-types/alloc", "mc-crypto-dalek/alloc"]
```

Ditto (conditional feature activation)
```suggestion
alloc = ["serde?/alloc", "mc-crypto-digestible/alloc", "mc-util-repr-bytes/alloc", "mc-transaction-types/alloc", "mc-crypto-dalek/alloc"]
```